test/components/testAddress.jsx

```diff
@@ -0,0 +1,188 @@
+/*
+ * testAddress.jsx - test the Address input component.
+ *
+ * Copyright © 2019, JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import enzyme, { mount, render } from 'enzyme';
+import PropTypes from 'prop-types';
+import Adapter from 'enzyme-adapter-react-16';
+
+import LocaleContext from '../../src/components/LocaleContext';
+import Address from '../../src/components/Address';
+import ilib from 'ilib-es6';
```

Comment:
It looks like there is nowhere to be used, isn't it ok to be deleted?

Reply:
The Address component is not merged or finished yet, and these tests don't even test anything real, so this file doesn't matter at the moment.